libvirt/src/virt_stats/__init__.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import json
+import sys
+
+import libvirt
+
+
+def libvirt_version():
+    version = libvirt.getVersion()
+    major = version // 1000000
+    version %= 1000000
+    minor = version // 1000
+    release = version % 1000
+
+    return "%d.%d.%d" % (major, minor, release)
+
+
+def hostname(domain: libvirt.virDomain):
+    try:
+        hostname = domain.hostname()
+    except libvirt.libvirtError:
+        macs = []
```

Comment:
Friendly reminder to add an extra print line here to explain that we are going to use an alternative approach to find the host name

Reply:
No need to, since we suppress the `stderr` output from `domain.hostname()` in https://github.com/tidalmigrations/machine_stats/pull/70/commits/83d9a8bf7949cc8541195231dbe43d7e04af9bc5